CODEOWNERS

```diff
@@ -276,6 +276,7 @@ build.json @home-assistant/supervisor
 /homeassistant/components/diagnostics/ @home-assistant/core
 /tests/components/diagnostics/ @home-assistant/core
 /homeassistant/components/digital_ocean/ @fabaff
+/tests/components/digital_ocean/ @fabaff
```

Comment:
Why do you add Fabian as codeowner? It's fine to add yourself though, you can do that in a separate PR named "Add phrfpeixoto as codeowner for digital_ocean"
```suggestion
```

Reply:
OK, maybe the check requires adding the same code owner as the implementation